pegasus/sites.v3/code.org/public/curriculum/generative-ai.haml

```diff
@@ -0,0 +1,131 @@
+    %p=hoc_s("gen_ai_page.course_description.desc")
+    = view "gen_ai_landing_page/empower_students_unordered_list"
+    .button-wrapper.centered
+      %a.link-button{href: CDO.studio_url("/courses/exploring-gen-ai-2024?viewAs=Instructor")}
+        =hoc_s(:call_to_action_explore_course)
+
+%section.bg-neutral-light
+  .wrapper
+    = view :curriculum_glance_section, heading: hoc_s("gen_ai_page.glance.title"), grades: hoc_s("gen_ai_page.glance.grades"), duration: hoc_s("gen_ai_page.glance.duration"), level: hoc_s("gen_ai_page.glance.level"), tools: hoc_s("gen_ai_page.glance.suggested_tools"), devices: hoc_s("gen_ai_page.glance.compatible_devices"), suggested_tools: hoc_s("gen_ai_page.glance.compatible_devices"), professional_learning: hoc_s("gen_ai_page.glance.professional_learning"), topics: hoc_s("gen_ai_page.glance.topics"), accessibility_features: hoc_s("gen_ai_page.glance.accessibility_features"), languages: hoc_s("gen_ai_page.glance.languages")
+
+%section
+  .wrapper
+    %h2
+      =hoc_s("gen_ai_page.teach_complete_course.title")
+    %p.body-two
+      =hoc_s("gen_ai_page.teach_complete_course.desc")
+    .action-block.action-block--two-col.content-block-margin
+      %img.col-50{src: "/images/ai/explore-gen-ai-bot-wizard.png", alt: ""}
+      .text-wrapper.col-50
+        %h3=hoc_s("gen_ai_page.teach_complete_course.explore_gen_ai_card.title")
+        %p=hoc_s("gen_ai_page.teach_complete_course.explore_gen_ai_card.desc")
+        %a.link-button{href: CDO.studio_url("/courses/exploring-gen-ai-2024?viewAs=Instructor")}
+          =hoc_s(:call_to_action_explore_course)
+        %br
+        %a.link-button.secondary.white-button{href: CDO.studio_url("/s/self-paced-gen-ai-2024")}
+          =hoc_s("gen_ai_page.teach_complete_course.explore_gen_ai_card.see_pl")
+      .clear
+
+= view :section_divider_line
+
+%section
+  .wrapper
+    %h2
+      =hoc_s("gen_ai_page.flexible_units.title")
+    %p.body-two
+      =hoc_s("gen_ai_page.flexible_units.desc")
+    = view "gen_ai_landing_page/generative_ai_units"
+
+%section.verified-skinny-banner
+  .wrapper.flex-container.justify-space-between.align-items-center.wrap.gap-1
+    .text-wrapper
+      %p.body-one.strong
+        =hoc_s("gen_ai_page.unlock_full_access_banner.title")
+      %p.body-two.no-margin-bottom
+        =hoc_s("gen_ai_page.unlock_full_access_banner.desc")
+    .cta-wrapper.col-30
+      %a.link-button.has-external-link{href: "https://support.code.org/hc/en-us/articles/115001550131-How-to-Become-a-Verified-Teacher"}
+        =hoc_s("gen_ai_page.unlock_full_access_banner.button")
+
+%section
+  .wrapper.ai-chat-lab.flex-container.justify-space-between.align-items-center
+    %figure.action-wrapper.solid-bg
+      %img{src: "/images/ai/model-card-genai-example.png", alt: ""}
+    .text-wrapper
+      %h2
+        =hoc_s("gen_ai_page.new_ai_chat_lab.title")
+      %p.body-two
+        =hoc_s("gen_ai_page.new_ai_chat_lab.desc")
```

Comment:
Nit: you can use [this view](https://docs.google.com/document/d/1wev8I2BQVViNXPuhLEXMpIAYSoAJb0kVoSRKaI3zEAk/edit#heading=h.6pqpkeqatzar) here instead of hard coding this. You'll just need to replace the `model-card-genai-example.png` image with a new export that includes the teal background.